Scenes/Towers/Projectiles/TemplateProjectile/TemplateProjectile.gd

```diff
@@ -19,4 +20,6 @@ func _physics_process(delta):
 	look_at(target.global_position)
 	var collision = move_and_collide(movement)
 	if collision:
+		if collision.collider is TemplateEnemy:
```

Comment:
Since the collision mask only has the enemy selected, I think we can skip this check, it should only collide with enemies. I want to rely on the engine to do things how we expect it to, and if it bugs out later and doesn't, then we can learn more by figuring out why

Reply:
Also a fun byproduct of this is if the projectile hits something other than the enemy it was aiming for, it will damage that enemy (the one it hit by mistake). Most games do not work like this but I kind of like it

Reply:
LOL that would be pretty funny if it did that. I can remove this check!

Reply:
- [ ] remove enemy check